pkg/constants/constants.go

```diff
@@ -248,7 +248,6 @@ var ResourcesToBlock = map[string]bool{
 	"namespacenetworkinfos.nsx.vmware.com":               true,
 	"ncpconfigs.nsx.vmware.com":                          true,
 	"network-attachment-definitions.k8s.cni.cncf.io":     true, // real name of NetworkAttachmentDefinition
```

Comment:
This should also be commented out.

Reply:
Also add comments on why we comment this out.

Reply:
Per our discussion, let's comment out the line above `real name of NetworkAttachmentDefinition` together.